plane/src/types/mod.rs

```diff
@@ -66,6 +70,14 @@ impl FromStr for ClusterName {
     }
 }
 
+impl TryFrom<String> for ClusterName {
+    type Error = &'static str;
+
+    fn try_from(value: String) -> Result<Self, Self::Error> {
+        Self::from_str(&value)
+    }
+}
+
```

Comment:
is there a reason you can't just call `ClusterName::from_str` in `database/backend.rs`?

Reply:
Not really, just syntactic sugar and seemed more uniform with the other types' `try_from` calls. I'll delete the function and call `from_str(&var)`